src/index.js

```diff
@@ -58,6 +58,9 @@ import handleStartupEvent from './squirrel';
     mainWindow = new BrowserWindow(generateBrowserConfig());
     global.mainWindowID = WindowManager.add(mainWindow, 'main');
 
+    // Required for 'darwin'
+    mainWindow.setFullScreenable(true);
```

Comment:
Think I found the issue: https://github.com/atom/electron/blob/v0.36.10/atom/browser/native_window_mac.mm#L490-L491

Looks like we need to set `{fullscreen: true}` as well (or in place of `fullscreenable`) in `src/main/configureBrowser.js` if we want it to Just Work without this line. I'm of the opinion that the app shouldn't default itself to fullscreen, but your call if you want to go that route. In that case this line can be removed.

Can try to patch electron such that `fullscreenable`can be set (via config options) despite `fullscreen` being `false` (and probably should override `maximizable` anyways). I'm the type of user that hates the maximize behavior on Mac apps in favor of "real" fullscreen, but there might be justification for having it the way it is. Otherwise this is probably **_the_** workaround, and maybe we can help them out with the docs.

Trying to poke around in https://github.com/atom/atom for reference into how they handle it, but haven't found it so far.